docs/multi_species.md

```diff
@@ -0,0 +1,34 @@
+# Multi-species models
+
+DeepForest allows training on multiple species annotations.
+When creating a deepforest model object, pass the designed number of classes and a label dictionary that maps each numeric class to a character label. 
+
+```
+m = main.deepforest(num_classes=2,label_dict={"Alive":0,"Dead":1})
+```
+
+It is often, but not always, useful to start from a prebuilt model when trying to identify multiple species. This helps the model focus on learning the multiple classes and not wasting data and time re-learning  bounding boxes.
```

Comment:
It is often, but not always, useful to start with a prebuilt model when trying to identify multiple species. This helps the model focus on learning the multiple classes and not waste data and time re-learning bounding boxes.